.github/actions/build-docker/action.yml

```diff
@@ -71,28 +61,25 @@ runs:
         echo "DOCKER_VERSION=${{ steps.meta.outputs.version }}" >> $GITHUB_ENV
         echo "DOCKER_COMMIT=${{ steps.context.outputs.git_sha }}" >> $GITHUB_ENV
         echo "DOCKER_BUILD=${{ steps.context.outputs.git_build_url }}" >> $GITHUB_ENV
+        echo "TAGS_FILE=${{ steps.meta.outputs.bake-file-tags }}" >> $GITHUB_ENV
+        echo "ANNOTATIONS_FILE=${{ steps.meta.outputs.bake-file-annotations }}" >> $GITHUB_ENV
+        echo "DOCKER_METADATA_FILE=buildx-bake-buildx-bake-buildx-bake-metadata.json" >> $GITHUB_ENV
```

Comment:
`buildx-bake-buildx-bake-buildx-bake-metadata.json` ? Why not just `buildx-bake-metadata.json` ?

Reply:
Good catch.